server/database/database.js

```diff
@@ -0,0 +1,47 @@
+import { Pool } from 'pg';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
```

Comment:
make sure you are switching between `development and testing databases` appropriately

Reply:
yes i have